.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
This file contain only bash comment, is it relevant to keep it ?

Reply:
Added lint-staged :)